packages/aws-cdk-lib/aws-cloudwatch-actions/test/lambda.test.ts

```diff
@@ -173,6 +178,8 @@ def handler(event, context):
     handler: 'index.handler',
   });
   alarm1.addAlarmAction(new actions.LambdaAction(alarmLambda));
+  alarm1.addOkAction(new actions.LambdaAction(alarmLambda));
+  alarm1.addInsufficientDataAction(new actions.LambdaAction(alarmLambda));
```

Comment:
```suggestion
  expect(() => {
    alarm1.addOkAction(new actions.LambdaAction(alarmLambda));
  }).toThrow(/There is already a Construct with name 'AlarmPermission' in Function \[alarmLambda\]/);
```
We should define a dedicated test case for this scenario.
`throws when multiple actions are created for the same alarm and lambda if feature flag is set to false`

Reply:
I'm a little confused about this comment, would you mind helping me understand what you are looking for? The line `alarm1.addOkAction(new actions.LambdaAction(alarmLambda));` will not throw in this case with the current implementation.